src/styles/main.scss

```diff
@@ -0,0 +1,65 @@
+@mixin circle($size) {
```

Comment:
It's not creating circle, but square.
Add border-radius rule to create circle.

Reply:
ready